### PR TITLE
Sigular and plural grammatical fix 

### PR DESCRIPTION
**Pull request by RockingThor**
Fixed the error message on the basis of the number. Also configured the tests accordingly. #3273

### CODE REVIEW

File: deno/lib/locales/en.ts [hunk 1]
```diff
@@ -70,11 +70,11 @@ const errorMap: ZodErrorMap = (issue, _ctx) => {
       if (issue.type === "array")
         message = `Array must contain ${
           issue.exact ? "exactly" : issue.inclusive ? `at least` : `more than`
-        } ${issue.minimum} element(s)`;
+        } ${issue.minimum} ${issue.minimum <= 1 ? "element" : "elements"}`;
```
Comment (m10rten) [inline deno/lib/locales/en.ts:73]: this is overkill, maybe do this instead:

```ts
message = `Array must contain ${
          issue.exact ? "exactly" : issue.inclusive ? `at least` : `more than`
        } ${issue.minimum} element${issue.minimum > 1 ? "s" : ""}`;
```

Comment (RockingThor) [reply deno/lib/locales/en.ts]: I have made the changes. Please have a look and let me know if any further changes are required.

Comment (m10rten) [reply deno/lib/locales/en.ts]: Looks better, it's up to maintainers to approve and merge this PR.

**Comment by colinhacks** [comment]
I appreciate this, though unfortunately a lot of users' tests are written to match Zod's exact error messages, so this would be considered a breaking change. I could merge this in Zod 4, but given Zod's current level of usership, I don't think the added error clarity is worth forcing users to refactor their test suites.

**Pull request closed without merging**